src/components/StaticAppBar/TopMenu.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import styled from 'styled-components';
+import Link from '../shared/Link';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import LINKS from './constants';
+import { connect } from 'react-redux';
+
+const NavLinkContainer = styled.div`
+  margin-left: 2rem;
+  display: block;
+  @media (max-width: 800px) {
+    display: none;
+  }
+`;
+
+const NavLink = styled(Link)`
+  padding: 0px 25px 7px;
+  margin: 0 2px;
+  text-transform: none;
+  word-spacing: 2px;
+  vertical-align: bottom;
+  color: white;
+  font-size: 1rem;
+  :hover {
+    font-weight: 600;
+    color: white;
+  }
+`;
+
+class NavMenu extends React.Component {
+  state = {
+    anchorEl: null,
+  };
+  handleMenu = event => {
+    this.setState({ anchorEl: event.currentTarget });
+  };
+
+  handleClose = () => {
+    this.setState({ anchorEl: null });
+  };
+
+  render() {
+    const { anchorEl } = this.state;
+    const open = Boolean(anchorEl);
+    const { url, label, sublinks = [] } = this.props.link;
+    const listItems = sublinks.map(({ label, url }) => (
+      <Link key={label} to={url}>
+        <MenuItem onClick={this.handleClose}>{label}</MenuItem>
+      </Link>
+    ));
+    const renderIcon = open ? (
+      <ExpandLess style={{ position: 'absolute' }} />
+    ) : (
+      <ExpandMore style={{ position: 'absolute' }} />
+    );
+    return (
+      <React.Fragment>
+        {!_.isEmpty(sublinks) ? (
+          <NavLink key={label} onClick={this.handleMenu}>
+            {label}
+            {renderIcon}
+          </NavLink>
+        ) : (
+          <NavLink key={label} to={url} onClick={this.handleMenu}>
+            {label}
+          </NavLink>
+        )}
+        {!_.isEmpty(sublinks) && (
+          <Menu
+            id="menu-appbar"
+            anchorEl={anchorEl}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'center',
+            }}
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'center',
+            }}
+            open={open}
+            onClose={this.handleClose}
+            getContentAnchorEl={null}
+            transitionDuration={0}
+          >
+            {listItems}
+          </Menu>
+        )}
+      </React.Fragment>
+    );
+  }
+}
+
+NavMenu.propTypes = {
+  link: PropTypes.object,
+};
+
+class TopMenu extends React.Component {
+  static propTypes = {
+    isAdmin: PropTypes.bool,
+    accessToken: PropTypes.string,
+  };
+  render() {
+    const { isAdmin, accessToken } = this.props;
+    const navLinks = LINKS.map(link => {
+      const checkRender = ({ label }) => {
```

Comment:
Do we really need a function `checkRender` ?

Reply:
I could do this directly, I thought there would be more conditional links at a later stage, so we can put it in `checkRender`